Routing.AspNetCore.Test/Routing.AspNetCore.Test.csproj

```diff
@@ -1,16 +1,17 @@
+<?xml version="1.0" encoding="utf-8"?>
 <Project Sdk="Microsoft.NET.Sdk">
-    <PropertyGroup>
-        <TargetFramework>netcoreapp3.0</TargetFramework>
-        <IsPackable>false</IsPackable>
-        <LangVersion>8.0</LangVersion>
-        <Nullable>enable</Nullable>
-        <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
-        <GenerateDocumentationFile>true</GenerateDocumentationFile>
-        <AssemblyName>Messerli.Routing.AspNetCore.Test</AssemblyName>
-        <RootNamespace>Messerli.Routing.AspNetCore.Test</RootNamespace>
-    </PropertyGroup>
-    <Import Project="..\.paket\Paket.Restore.targets" />
-    <ItemGroup>
-        <ProjectReference Include="..\Routing.AspNetCore\Routing.AspNetCore.csproj" />
-    </ItemGroup>
-</Project>
+  <PropertyGroup>
```

Comment:
Why has the indentation changed?

Reply:
Addressed in: https://github.com/messerli-informatik-ag/routing/pull/22/commits/fd7ef97fd812dbd35244eb994aacadd05b96f611